src/tools/lib/python/energyHistogram.py

```diff
@@ -0,0 +1,187 @@
+"""
+This file is part of PIConGPU.
+
+PIConGPU is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+PIConGPU is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with PIConGPU.
+If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2017 Marco Garten, Axel Huebl
+Authors: Axel Huebl
+License: GPLv3+
+"""
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import h5py
+
+
+class EnergyHistogram:
+    """
+    Class to help plotting data fromt the energyHistogram plugin.
+    """
+    def __init__(
+            self,
+            species_name="ph",
+            sim="./",
+            period=1000,
+            bin_count=1024,
+            min_energy=10,
+            max_energy=10000,
+            distance_to_detector=0,
+            slit_detector_x=None,
+            slit_detector_z=None):
```

Comment:
The intention was to have them available and maybe even deduce them from the PIConGPU input files so that further analysis could be done with them. 
Calculating an opening angle from the slit size and the distance for example.
But I agree, either they should be removed now or some additional functionality should be implemented. I cannot think of anything useful that is general enough at the moment that would justify keeping them, so I'll remove them.